src/components/SwapSettingsModal/SwapSettingsModal.tsx

```diff
@@ -95,7 +98,7 @@ const SetSlippage: React.FC<{ closeModal: () => void }> = ({ closeModal }) => {
   });
 
   /* SLIPPAGE */
-  const [inputFocused, setInputFocused] = useState(!slippageInitialPreset);
```

Comment:
curious what was the bad state before

Reply:
Down to race condition between onchange, and react's batching of renders